api/transactions.py

```diff
@@ -1,25 +1,21 @@
 from decimal import Decimal
 from typing import List
-from typing import Union
 
 from fastapi import APIRouter
-from fastapi import Depends
 from fastapi import HTTPException
 
-from core.security import get_api_key
-from models.transaction import Transaction
+from schemas.transactions import Transaction
 from services.actual_service import actual_service
 
 router = APIRouter()
 
 
-@router.post("/transactions", dependencies=[Depends(get_api_key)])
```

Comment:
Removed as `/transaction` and `/transactions/` will work regardless, no need to add both.
Additionally we do not need the dependency on `get_api_key` here as it's already added in `main.py` when we add the route

Reply:
Hmm, I realized this doesn't work actually when testing with Ios shortcuts. Yes `/transactions` and `/transactions/` work in postman but after merging this PR it seems to have broken the original shared IOS shortcut which uses `/transactions/` I also realized you removed `dependencies=[Depends(get_api_key)])` which we would want to keep in order to protect the endpoint with the api.